install_config/install/host_preparation.adoc

```diff
@@ -245,7 +245,8 @@ Installation] method, you can easily script a complete installation from a
 kickstart or cloud-init setup, change the default configuration file:
 +
 ----
-# sed -i '/OPTIONS=.*/c\OPTIONS="--selinux-enabled --insecure-registry 172.30.0.0/16"' \
+# sed -i '/OPTIONS=.*/c\OPTIONS="--selinux-enabled --insecure-registry
+# 172.30.0.0/16"' --signature-verification=false'\
```

Comment:
@tmorriso-rh I think this should be on one line. 
sed -i '/OPTIONS=.*/c\OPTIONS="--selinux-enabled --insecure-registry 172.30.0.0/16"' --signature-verification=false'

Putting  `# 172.30.0.0/16"' --signature-verification=false'\` on a separate line suggests entering `172.30.0.0/16"' --signature-verification=false'\` as a separate command.

To avoid scrolling text boxes, you can split the command with the `\` character:
`sed -i '/OPTIONS=.*/c\OPTIONS="--selinux-enabled \
 --insecure-registry 172.30.0.0/16"' --signature-verification=false'\`  

Maybe double-check with someone on the team to make sure i have this right.

Reply:
I believe so. So judging by the BZ request, and keeping it all on "one line", I'd say:

# sed -i '/OPTIONS=.*/c\OPTIONS="--selinux-enabled \
  --insecure-registry 172.30.0.0/16" \
  --signature-verification=false' \ /etc/sysconfig/docker

I'm not too sure of the extra \ after "false". That might need to be followed up on (I'm not familiar with using the sed command). @adellape or @ahardin-rh might understand it more.

Reply:
Thanks @mburke5678 and @bfallonf. Can you take a look at the updated file?

Reply:
Hmm @tmorriso-rh I can't see any changes. Did you push them back to Github?

Reply:
Hi Brice, I had some trouble pushing the changes back to GitHub, but you should be able to see the changes now. David, the bug reporter, recommended not using line continuations and allowing the preformatted area to scroll. Thanks.